RELEASE-NOTES.md

```diff
@@ -4,6 +4,95 @@ section: overview
 releaseNoteTOC: true
 ---
 
+## 2020.01 release notes (2020-01-28)
+Packages released:
+
+
+### Components
+- **About modal:** 
+  - Added type safety to default props ([#3536](https://github.com/patternfly/patternfly-react/pull/3536))
+- **Actions column:** 
+  -  Fixed import path for DropdownSeparator ([#3578](https://github.com/patternfly/patternfly-react/pull/3578))
+- **Alert:** 
+  - Accessibility updates to alert for toast alerts ([#3519](https://github.com/patternfly/patternfly-react/pull/3519))
+- **Chip group:**
+  - Added closable chip group ([#3455](https://github.com/patternfly/patternfly-react/pull/3455))
+- **Clipboard copy:** 
+  - Set type of copy and toggle buttons to 'button' ([#3444](https://github.com/patternfly/patternfly-react/pull/3444))
+- **Data list:** 
+  - Properly assigned selectable and selected classes ([#3447](https://github.com/patternfly/patternfly-react/pull/3447))
+  - Removed hook ([#3434](https://github.com/patternfly/patternfly-react/pull/3434))
+  - Reverted React.Component back to React.FunctionComponent ([#3431](https://github.com/patternfly/patternfly-react/pull/3431))
+- **Data toolbar:** 
+  - Added and removed some modifiers to match core ([#3422](https://github.com/patternfly/patternfly-react/pull/3422))
+  - Reverted promotion of DataToolbar from experimental ([#3580](https://github.com/patternfly/patternfly-react/pull/3580))
+  - Updated example to not exclude previous filtered items. ([#3571](https://github.com/patternfly/patternfly-react/pull/3571))
+- **Divider:** 
+  - Promote Divider from experimental ([#3516](https://github.com/patternfly/patternfly-react/pull/3516))
+- **Flex layout:** Allowed string literals to be used for FlexItem modifiers ([#3544](https://github.com/patternfly/patternfly-react/pull/3544))
```

Comment:
Shouldn't this be under `### Layouts`?

Reply:
I debated that... I did not see a layout section (unless I missed it) in previous release notes where a layout was updated and I wanted to be consistent.

Reply:
It looks like we are not consistent about this. Core currently has a `### Layouts` as well as https://www.patternfly.org/v4/documentation/core/overview/release-notes `**Flex layout:**`. I'm okay with either after seeing that!